src/index.html

```diff
@@ -1,16 +1,31 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="server.name"
+      method="get"
+    >
+      <label class="label">
+        <input
+          type="search"
+          placeholder="Try &#34Los Angeles&quot"
+          data-qa="big"
+          class="search search__big"
+        >
+        <input
+          type="search"
+          placeholder="Try &#34Los Angeles&quot"
+          data-qa="small"
+          class="search search__small"
+        >
+      </label>
+    </form>
```

Comment:
in this case with search bars, there should be two forms (+ don't forget about `data-qa` attributes to make sure it is possible to auto-test you solution)
```
form --big
	label
		input

form --small
	label
		input